packages/components/molecules/f-restaurant-card/src/components/_tests/RestaurantCard.v1.test.js

```diff
@@ -236,4 +236,32 @@ describe('RestaurantCard.v1', () => {
             expect(wrapper.find('[data-test-id="restaurant-delivery-time-meta"]').exists()).toBe(true);
         });
     });
+
+    describe('Offer', () => {
+        it('displays an offer if one is provided', () => {
+            // arrange
+            const propsData = {
+                offer: 'foo bar baz'
+            };
+
+            // act
+            const wrapper = mount(RestaurantCardV1, { propsData });
+
+            // assert
+            expect(wrapper.find('[data-test-id="restaurant-offer"]').exists()).toBe(true);
+        });
+
+        it('does not display an offer if none exists', () => {
```

Comment:
Worth checking undefined and an empty string?

Reply:
undefined, sure. empty string - not so concerned. I don't think it's a likely scenario and I'd prefer not to call a trim method on every restaurant offer

Reply:
sorry - misread as a string with just spaces. I will add an empty string one.